chart/values.schema.json

```
@@ -1897,7 +1897,7 @@
         },
         "stores": {
           "$ref": "#/$defs/EnableSwitch",
-          "description": "Stores defines if secret stores should get synced from the virtual cluster to the host cluster."
+          "description": "Stores defines if secret stores should get synced bi-directionally after an initial sync from the virtual cluster to the host cluster."
```

This phrasing is slightly confusing, it sounds like you are defining the bi-directional part only. How about: 
```suggestion
          "description": "Stores defines if secret stores should get synced from the virtual cluster to the host cluster and then bi-directionally."
```